src/mbgl/annotation/annotation_manager.cpp

```diff
@@ -166,13 +159,26 @@ void AnnotationManager::updateStyle(Style& style) {
         shape.second->updateStyle(style);
     }
 
+    for (const auto& image : images) {
+        if (!style.getImage(image.first)) {
+            style.addImage(std::make_unique<style::Image>(image.second));
```

Comment:
Could we move them instead of copying them? What is the benefit of retaining the image?